pkg/gatewayserver/gatewayserver_test.go

```diff
@@ -62,7 +62,7 @@ var (
 	unregisteredGatewayID  = "eui-bbff000000000000"
 	unregisteredGatewayEUI = types.EUI64{0xBB, 0xFF, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
 
-	timeout        = (1 << 5) * test.Delay
+	timeout        = (1 << 7) * test.Delay
```

Comment:
Multiply the `timeout` selectively, not GS wide

Reply:
Rolled back